backend/src/services/GenerateReporter/GenerateReportService.ts

```diff
@@ -70,6 +71,15 @@ export class GenerateReportService {
   async generateReporter(
     request: GenerateReportRequest
   ): Promise<GenerateReporterResponse> {
+
+    if (Object.keys(request).length === 0) {
+      throw new LackRequiredDataError();
```

Comment:
It’s best to indicate which field is missing

Reply:
It is indicated in the LackRequiredDataError.ts, says "Lack Required Data". It is a bit of completed to judge which specific field is not provided so I chose this way.